scripts/Run-Tests.ps1

```diff
@@ -8,12 +8,25 @@ param (
     $Action = 'Bench',
 
     [string]
-    $Note = '',
+    $Note,
 
     [string]
     $OutDirectory = '.\test\results',
 
+    [string]
+    $TestDirectory = '.\bin\test',
+
+    [ValidateSet('Functional', 'CRI')]
+    [string]
+    $TestExe = 'CRI',
```

Comment:
Could we not just pass through the full path to these instead of needing to figure that out later?

Reply:
we could, but the default case would then require writing out the full path, instead of having two options that pwsh tab-completes for you
plus, im assuming that people arent naming their binaries something other than `<pkg>.test.exe`, since most shouldnt

Reply:
I don't think this is blocking for now since this is a test script and we can easily change this in the future. Resolving for now.